resources/assets/js/app/models/UserModel.js

```diff
@@ -1,5 +1,9 @@
 var baseModel = require('../instances/Model');
 
 module.exports = baseModel.extend({
-    urlRoot: '/users'
+    urlRoot: '/users',
+    initialize: function (data, options) {
+        if(options)
+            if(options.parentUrl) this.parentUrl = options.parentUrl;
```

Comment:
implement this logic in BaseModel, we have a lot of same initializers accross app